code/modules/vore/eating/belly_obj_vr.dm

```diff
@@ -1991,7 +1993,10 @@
 		else
 			playsound(src, struggle_rustle, vary = 1, vol = 75, falloff = VORE_SOUND_FALLOFF, frequency = noise_freq, preference = /datum/preference/toggle/digestion_noises, volume_channel = VOLUME_CHANNEL_VORE) //CHOMPEdit
 	//CHOMPEdit End
-
+	//CHOMPEdit Start
```

Comment:
End and start following each other, remove those two comments and keep the section going

Reply:
Got it, should be fixed now.